internal/infrastructure/db/migrations/20240819203419_create_tables_v1.sql

```diff
@@ -4,49 +4,68 @@ SELECT 'up SQL query';
 
 SET search_path = events_api, public;
 
+CREATE TABLE IF NOT EXISTS developer_licenses (
+      license_address BYTEA PRIMARY KEY,
+      developer_id CHAR(27) NOT NULL,
+      status TEXT NOT NULL CHECK (status IN ('Active', 'Inactive')),
+      created_at TIMESTAMP WITH TIME ZONE NOT NULL DEFAULT CURRENT_TIMESTAMP,
+      updated_at TIMESTAMP WITH TIME ZONE NOT NULL DEFAULT CURRENT_TIMESTAMP
+);
+
 CREATE TABLE IF NOT EXISTS events (
-                                      id char(27) NOT NULL,
-                                      condition TEXT NOT NULL,
-                                      flow_type TEXT NOT NULL CHECK (flow_type IN ('Realtime', 'Batch')),
-                                      event_name TEXT NOT NULL,
-                                      webhook_target_uri TEXT NOT NULL,
-                                      cooldown_period INTEGER NOT NULL DEFAULT 0,
-                                      created_at TIMESTAMP WITH TIME ZONE NOT NULL DEFAULT CURRENT_TIMESTAMP,
-                                      updated_at TIMESTAMP WITH TIME ZONE NOT NULL DEFAULT CURRENT_TIMESTAMP,
-                                      status TEXT NOT NULL CHECK (status IN ('Active', 'Inactive')),
-                                      owner_dev_license_address BYTEA NOT NULL,
-
-                                      CONSTRAINT events_pkey PRIMARY KEY (id)
+      id CHAR(27) NOT NULL,
+      event_name TEXT NOT NULL,
+      condition JSONB NOT NULL DEFAULT '{}'::JSONB,
```

Comment:
i think we'll want to tweak where these conditions live, but good to merge now and we can always change it later